src/widgets/openmediavault/OpenMediaVaultTile.tsx

```diff
@@ -0,0 +1,119 @@
+import { Box, Flex, Group, Paper, ScrollArea, Text } from '@mantine/core';
+import { IconInfoSquare } from '@tabler/icons-react';
+import { useTranslation } from 'next-i18next';
+import { useConfigContext } from '~/config/provider';
+import { api } from '~/utils/api';
+
+import { defineWidget } from '../helper';
+import { WidgetLoading } from '../loading';
+import { IWidget } from '../widgets';
+import OpenMediaVaultCpu from './OpenMediaVaultCpu';
+import OpenMediaVaultFileSystem from './OpenMediaVaultFileSystem';
+import OpenMediaVaultMemory from './OpenMediaVaultMemory';
+
+const definition = defineWidget({
+  id: 'openmediavault',
+  icon: 'https://cdn.jsdelivr.net/gh/walkxcode/dashboard-icons/png/openmediavault.png',
```

Comment:
Since the name of the widget needs to be general, I suggest this icon to represent the widget:
https://tabler.io/icons/icon/heart-rate-monitor
import { IconHeartRateMonitor } from '@tabler/icons-react'